integration_tests/__tests__/failures-test.js

```diff
@@ -17,6 +17,25 @@ const normalizeDots = text => text.replace(/\.{1,}$/gm, '.');
 
 skipOnWindows.suite();
 
+const cleanupStackTrace = stderr => {
```

Comment:
we can't run both versions yet (jasmine and circus). And this is mostly for me. when i'm done with circus this will be removed, and the error is there only to make sure i don't forget to remove it after i'm done :)